### DISCUSSION TITLE
How do high school students apply for the github student pack?

### DISCUSSION THREAD

**Discussion by xycld** [community/community#51492] [GitHub Education]
I am a high school student, located in China, and I want to apply for the github student pack, but my application has been rejected many times."You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio."I tried many things, including translating my acceptance letter documents and changing my profile, but they didn't work.Our school does not provide educational emails, I hope someone can help me, thank you very much!

**Comment by samus-aran** [comment]
Hi @xy-cloud-cn  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by xycld** [reply]
![image](https://user-images.githubusercontent.com/85053859/229019523-2a6cbc88-3860-4fd1-b9b7-e9ce9457df33.png)
thx

**Reply by jge162** [reply]
You need to complete your profile, it currently looks like a gamer_tag or basic username. Put your first name and last name instead. @xy-cloud-cn good luck.

**Reply by xycld** [reply]
I actually changed my name, but that didn't work and I'm curious if this was a machine review and it may not have recognized my name on my acceptance letter

**Reply by jge162** [reply]
my profile has my name and it approved in less than 24 hours.

**Reply by rilysh** [reply]
> I actually changed my name, but that didn't work and I'm curious if this was a machine review and it may not have recognized my name on my acceptance letter

Did you put your *real* name there? Also, what did you see after you change your name there? And no, GitHub controls student verification through staff reviews.

**Reply by jge162** [reply]
I put my real name in description and bio

**Reply by xycld** [reply]
> > I actually changed my name, but that didn't work and I'm curious if this was a machine review and it may not have recognized my name on my acceptance letter
> 
> Did you put your _real_ name there? Also, what did you see after you change your name there? And no, GitHub controls student verification through staff reviews.

It also asked me to change my profile after I changed my name, and I wasn't rejected after a while, but directly after submitting

**Reply by xycld** [reply]
> I put my real name in description and bio

Yes, that's what I did.

**Reply by jge162** [reply]
Idk then.

**Reply by xycld** [reply]
> Idk then.

ok thx

**Reply by xycld** [reply]
The problem has not been resolved, I still need help, thanks

**Comment by xycld** [comment]
Ok, there is progress on the question, why my application was rejected directly after I clicked the button, while you waited a long time for a reply

**Reply by samus-aran** [reply]
Hi @xy-cloud-cn 

Can't comment on the immediate application status as this can vary during busy periods.

Regards to updating your profile information we would suggest that after you have updated your profile that you log out and log back into GitHub before trying to re-apply. Can you do this and then let us know how you get on 👍

**Reply by xycld** [reply]
> Hi @xy-cloud-cn
> 
> Can't comment on the immediate application status as this can vary during busy periods.
> 
> Regards to updating your profile information we would suggest that after you have updated your profile that you log out and log back into GitHub before trying to re-apply. Can you do this and then let us know how you get on 👍

I tried, but at the end of the progress bar, he popped up this message directly
![image](https://user-images.githubusercontent.com/85053859/229292599-99c4c4b1-3e97-476d-bc23-86952ff5dce9.png)

**Reply by xycld** [reply]
> Hi @xy-cloud-cn
> 
> Can't comment on the immediate application status as this can vary during busy periods.
> 
> Regards to updating your profile information we would suggest that after you have updated your profile that you log out and log back into GitHub before trying to re-apply. Can you do this and then let us know how you get on 👍

hi?Is there any other solution? Or, for my case, is it possible to request a manual review, I really need the github student pack, thanks again!

**Reply by xycld** [reply]
Hi @samus-aran 
Do you have any more solutions?

**Comment by xycld** [comment]
who can help me😥

**Comment by gladdduck** [comment]
i have the same problem ,and it still  exist

**Reply by xycld** [reply]
But you're a university student, you can try the 学信网 information and add your name to the bio, then try a few times
I'm rather interested to know if you received a rejection message directly after the pre-processing, if so, that might give me some help, thx

**Comment by jamesopppp** [comment]
i have the same problem ,and it still exist

**Reply by xycld** [reply]
so many Chinese student，sad
